src/dbus/experimental.rs

```diff
@@ -0,0 +1,42 @@
+//! Experimental interface.
+
+use crate::update_agent::{LastRefresh, UpdateAgent};
+use actix::Addr;
+use futures::prelude::*;
+use tokio::runtime::Runtime;
+use zbus::{dbus_interface, fdo};
+
+/// Experimental interface for testing.
+pub(crate) struct Experimental {
+    pub(crate) agent_addr: Addr<UpdateAgent>,
+}
+
+#[dbus_interface(name = "org.coreos.zincati.Experimental")]
+impl Experimental {
+    /// Just a test method.
+    fn moo(&self, talkative: bool) -> String {
+        if talkative {
+            String::from("Moooo mooo moooo!")
+        } else {
+            String::from("moo.")
+        }
+    }
+
+    /// Get update_agent actor's last refresh time.
+    fn last_refresh_time(&self) -> fdo::Result<i64> {
+        let msg = LastRefresh {};
+        let refresh_time_fut = self.agent_addr.send(msg).map_err(|e| {
+            let err_msg = format!("failed to get last refresh time from agent actor: {}", e);
+            log::error!("LastRefreshTime D-Bus method call: {}", err_msg);
+            fdo::Error::Failed(err_msg)
+        });
+
+        Runtime::new()
```

Comment:
Following up to this comment https://github.com/coreos/zincati/pull/514#discussion_r605931720, I've opted to use tokio's `Runtime::new()` directly.

Reply:
Ah, I think you are right. I was hoping to reuse the existing `System` but it looks like it can't be done that way.
There is possibly a smarter way to share the runtime here (without creating a new one from scratch each time), but this looks good enough for now.

Reply:
Coming back to this after some additional reading on async and actix: 
Is the reason why we can't use the existing `System` because the DBus actor is hosted on a separate `SyncArbiter`, which is not async? Notably, we don't have access to an `AsyncContext`. It looks to me like there is no executor for this actor, so we have to create our own.
For some context, the DBus actor is started via `SyncArbiter::start(threads, move || DBusService::new(agent_addr.clone()))`